dashboard/app/models/unit_group.rb

```diff
@@ -326,10 +326,10 @@ def assignable_for_user?(user)
   # A course that the general public can assign. Has been soft or
   # hard launched.
   def launched?
-    [SharedConstants::PUBLISHED_STATE.preview, SharedConstants::PUBLISHED_STATE.stable].include?(published_state)
+    [SharedConstants::PUBLISHED_STATE.preview, SharedConstants::PUBLISHED_STATE.stable].include?(get_published_state)
   end
 
-  def published_state
+  def get_published_state
```

Comment:
Update `if is_stable` to check `if stable?` so that we set published state on plc courses and course without family names which have a special case to treat as stable right now